### PR TITLE
fix: Execute fill with updated message and recipient

### PR DESCRIPTION
**Pull request by nicholaspai**
When sped up, updated message and recipient are not used

### CODE REVIEW

**Review by mrice32 — APPROVED** [review]
Very nice find!

File: contracts/SpokePool.sol [hunk 1]
```diff
@@ -1223,24 +1223,28 @@ abstract contract SpokePool is
...
                     msg.sender,
-                    relayData.recipient,
+                    relayExecution.updatedRecipient,
+                    amountToSend
+                );
+            else
+                IERC20Upgradeable(relayData.destinationToken).safeTransfer(
+                    relayExecution.updatedRecipient,
                     amountToSend
                 );
```
Comment (james-a-morris) [inline contracts/SpokePool.sol:1240]: Nit: should we add the `tsconfig` setting to require brackets after conditionals?

**Review by james-a-morris — COMMENTED** [review]
Left a comment

File: contracts/SpokePool.sol (hunk 1 above)
Comment (nicholaspai) [reply contracts/SpokePool.sol:1240]: do you know which config we should set for that? I marginally favor requiring brackets but not a big deal

**Pull request merged**